src/vs/workbench/services/suggest/browser/simpleCompletionModel.ts

```diff
@@ -182,8 +182,16 @@ export class SimpleCompletionModel {
 		}
 
 		this._filteredItems = target.sort((a, b) => {
-			// Sort first by the score
-			let score = b.score[0] - a.score[0];
+			// Keywords should always appear at the bottom when they are not an exact match
+			let score = 0;
+			if (a.completion.isKeyword && a.labelLow !== wordLow || b.completion.isKeyword && b.labelLow !== wordLow) {
+				score = (a.completion.isKeyword ? 1 : 0) - (b.completion.isKeyword ? 1 : 0);
```

Comment:
Is this score referring to the indexing of suggestion? For example, would low score represent first/last thing that would show up as a suggestion

Reply:
@anthonykim1 this is fed into the a sort function such that `exit` in pwsh has special sorting behavior as keywords are generally unlikely to be used in the repl